tests/kani/AsyncAwait/manual_spawn.rs

```diff
@@ -60,60 +110,98 @@ pub struct RoundRobin {
 }
 
 impl SchedulingStrategy for RoundRobin {
```

Comment:
can you use the `kani::RoundRobin` instead?

Reply:
I thought it would be nice to have a manual implementation (not from the library) in `manual_spawn.rs` because there is a speed difference (#1876). But I can remove the file (since it duplicates code from the Kani library) if you want.

Reply:
You can add manual tests, but I would avoid a complete duplicate.

Reply:
I'm not sure what else to test manually, so I'll remove it.